draft-ietf-tls-tls13.md

```diff
@@ -2261,13 +2261,15 @@ The "extension_data" field of this extension contains an
 
 %%% Key Exchange Messages
 
+       struct {} None;
+
        struct {
            select (Handshake.msg_type) {
                case client_hello:
                    uint32 obfuscated_ticket_age;
 
                case server_hello:
-                   struct {};
+                   None;
```

Comment:
If we used this notation in more places, maybe, however just doing things like this feels like excessive indirection. A simpler simplification would be to just drop the word "struct" and state the payload as "{}". I don't feel like this is a particularly useful thing to fiddle with, however. It's fine as-is.